playbooks/roles/foreman_repositories/tasks/debian_release_repos.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: 'Install Foreman GPG key'
+  apt_key:
+    url: https://deb.theforeman.org/foreman.asc
```

Comment:
Using an HTTPS url here requires `ca-certificates` to be installed. this is the case on the default vagrant debian box, but may be not on others. May I suggest adding the following?

```yaml
- name: install ca-certificates
  apt:
    name: ca-certificates
    state: present
```

before this task?